.gitignore

```diff
@@ -4,4 +4,6 @@ db.json
 *.log
 node_modules/
 public/
-.deploy*/
\ No newline at end of file
+.deploy*/
+
+source/_drafts
```

Comment:
I think this should be in the source control

Reply:
Why? I see tradeoffs on either including them or removing it.

If we include them, then our PR's will have data that we don't want to be reviewed.
If you exclude them, then we loose our ability to work on different computers given that they will be ignored.

Reply:
What worries is to repeat the mistake I did, I accidentally removed a post and it wasn't in the source control u.u

Maybe we should not add draft posts to the PRs?

Reply:
Yeah, we can keep _drafts in source control and don't add them to PR's if we see is a PITA to keep track we can later remove it.